frontend/src/components/CreateCategoryModal.svelte

```diff
@@ -173,19 +175,28 @@
         </div>
         <div class="flex flex-row justify-between items-center">
             <Heading size="md" tag="h4">Minimum Score</Heading>
-            <input aria-label="min_score" bind:value={minScore} class="w-full rounded-lg" required type=number>
+            <input aria-label="min_score" bind:value={minScore} max={maxScore - 1} class="w-full rounded-lg" required type=number>
         </div>
+        {#if minScore >= maxScore}
+            <Helper class="mt-2 text-red-500" visable={false}><span class="font-medium text-red-500">Warning!</span>
+                Minimal Score can not be higher or equal the Maximal Score
+            </Helper>
+        {/if}
         <div class="flex flex-row justify-between items-center">
             <Heading size="md" tag="h4">Maximum Score</Heading>
-            <input aria-label="max_score" bind:value={maxScore} class="w-full rounded-lg" required type=number>
+            <input aria-label="max_score" bind:value={maxScore} min={minScore} class="w-full rounded-lg" required type=number>
         </div>
         <div class="flex flex-row justify-between items-center">
             <Heading size="md" tag="h4">Score Step Size</Heading>
-            <input aria-label="score_step_size" bind:value={scoreStepSize} class="w-full rounded-lg" required type=number>
+            <input aria-label="score_step_size" bind:value={scoreStepSize} max={maxScore-minScore} class="w-full rounded-lg" required type=number>
+
         </div>
-        <Button color="primary"
-                on:click={() => {finishCreation();createCategory();}}
-                size="xs" type="submit">
+        {#if scoreStepSize > maxScore - minScore}
+            <Helper class="mt-2 text-red-500" visable={false}><span class="font-medium text-red-500">Warning!</span>
+                Score Step Size can not be higher that the difference of Minimal and Maximal Score
```

Comment:
The score step size cannot be greater than the difference between the minimum and maximum scores.